spec/controllers/restaurant_controller_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+RSpec.describe RestaurantsController, type: :controller do
+  let(:restaurant_hash) do
+    {
+      name: 'Name',
+      genre: :sushi,
+      rating: 1,
+      accepts_10bis: true,
+      delivery_time: :slow
+    }
+  end
+  let(:keys) { restaurant_hash.keys.to_a }
+  before (:each) do
+    Restaurant.create(restaurant_hash)
+  end
+
+  describe 'validations' do
+    it 'should allow creating a restaurant' do
+      new_hash = restaurant_hash.clone
+      new_hash[:name] = 'New Name'
+      post :create, params: { restaurant: new_hash }, format: :json
+      expect(response).to be_success
+
+      get :index, format: :json
```

Comment:
This was a way to test the creation of the mode, but your previous comment is correct and makes this redundant